llvm.spec

```diff
@@ -48,12 +45,8 @@ cmake %{_builddir}/llvm-%{realversion}-%{llvmCommit}/llvm \
   -DLLVM_ENABLE_EH:BOOL=ON \
   -DLLVM_ENABLE_PIC:BOOL=ON \
   -DLLVM_ENABLE_RTTI:BOOL=ON \
-%ifarch x86_64
   -DLLVM_TARGETS_TO_BUILD:STRING="X86;PowerPC;AArch64;NVPTX" \
   -DLIBOMPTARGET_NVPTX_ALTERNATE_HOST_COMPILER=/usr/bin/gcc \
```

Comment:
Note to self: should we add something like
```spec
%ifarch x86_64 ppc64le
  -DLIBOMPTARGET_NVPTX_COMPUTE_CAPABILITIES="sm_35;sm_60;sm_70" \
%endif
%ifarch aarch64
  -DLIBOMPTARGET_NVPTX_COMPUTE_CAPABILITIES="sm_70;sm_72" \
%endif
```
(or just always all of them) ?

Reply:
why not
```
%ifnarch aarch64
  -DLIBOMPTARGET_NVPTX_COMPUTE_CAPABILITIES="sm_35;sm_60;sm_70" \
%else
  -DLIBOMPTARGET_NVPTX_COMPUTE_CAPABILITIES="sm_70;sm_72" \
%endif
```

Reply:
I mildly prefer to keep the list of architectures explicit, so we know what we build for what arch

(and, we don't get errors if we add a new, unsupported architecture like RISC-V)

Reply:
an explict list needs to be in sync with https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_1_X/master/cuda-flags.file right?

Reply:
anyway, since LIBOMPTARGET does not depend on the host but only on the device, it's simpler if we list all devices we want unconditionally

Reply:
we can include `cuda-flags` here and get the list of devices

Reply:
> an explict list needs to be in sync with https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_1_X/master/cuda-flags.file right?

not really: this is used to tell clang what devices we want to support offloading to using Open
MP 4 - which we currently don't use; I think we are fine as long as this list is a superset of the devices we build CUDA code for.

> we can include `cuda-flags` here and get the list of devices

Good point, let me see if I can come up with a decent syntax.